.github/workflows/ci_cron_weekly.yml

```diff
@@ -54,3 +54,24 @@ jobs:
         python -m pip install tox
     - name: Test with tox
       run: tox -e py311-test-devdeps-romandeps
+
+  ci_cron_tests_stable_strauss:
+    name: Python 3.12 with stable versions of dependencies and Strauss
+    runs-on: ubuntu-latest
+    if: (CI == false && (github.repository == 'spacetelescope/jdaviz' && (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' || contains(github.event.pull_request.labels.*.name, 'Extra CI'))))
```

Comment:
Now that I see this `if` here, it does not quite make sense to add a GitHub Actions job and then tell it to never run. Options:

1. If you still want other tests to run and just to make sure non-strauss stuff does not break when strauss is installed, only skip the failing tests when CI is true. Do not skip the whole job.
2. If (1) means nothing to you, just remove this whole job from this YAML. It is dead code at this point.

Reply:
Do I put that skip in the actual test file `test_sonify_plugin.py` or in this file?

Reply:
that effectively makes the test useless though, right (unless we occasionally remember to run locally).  Is it possible to test that the plugin is disabled because of no sound devices but then still call the internals to generate the sonified cube and assert its contents without ever playing it?

Reply:
Similar thoughts as Kyle (but less clear) at https://github.com/spacetelescope/jdaviz/pull/3269#discussion_r1884037759

Reply:
It would require a non-trivial amount of code moving that should be covered by #3330.

Reply:
`vue_sonify_cube()` calls `flux_viewer.get_sonified_cube()` which requires a device ID because that method creates an instance of `CubeListenerData` called `audified_cube`, which does not need a device ID. However, later in the method the `sounddevice.OutputStream` is created which does need the device ID. I could separate that stream creation from the `get_sonified_cube` method but that work may be overwritten by #3330 . So is that work worth it to do in this PR if we may do something different down the road? If the answer is yes then I can do that but I wanted to be clear with why I think that effort is covered by #3330.

Reply:
Given the rush, probably okay to defer unless you think that refactoring is critical here for AAS.